src/cascadia/UIHelpers/Converters.cpp

```diff
@@ -58,7 +58,14 @@ namespace winrt::Microsoft::Terminal::UI::implementation
     // Misc
     winrt::Windows::UI::Text::FontWeight Converters::DoubleToFontWeight(double value)
     {
-        return winrt::Windows::UI::Text::FontWeight{ base::ClampedNumeric<uint16_t>(value) };
+        uint16_t val = 400;
```

Comment:
this looks like an unrelated change - but I see it removes `base::ClampedNumeric` so I get it

Reply:
Yeah, this got in accidentally, as I developed this simultaneously with my other branch that removes the chromium library. I left it in because I felt like it still sort of fits the theme of the PR.